misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -425,4 +428,26 @@ class FakeRedis : Redis {
   override fun zscore(key: String, member: String): Double? {
     throw NotImplementedError("Fake client not implemented for this operation")
   }
+
+  override fun zrange(
+    key: String,
+    type: ZRangeType,
+    start: ZRangeMarker,
+    stop: ZRangeMarker,
+    reverse: Boolean,
+    limit: ZRangeLimit?,
+  ): List<ByteString?> {
+    throw NotImplementedError("Fake client not implemented for this operation")
```

Comment:
FakeRedis in testing seems to support, should the fake client also support this method?

Reply:
FakeRedis is deprecated. None of the methods added after March 2023 are implemented here. 
```
@Deprecated(
  message = "Moved to misk-redis-testing. This Fake does not emulate redis correctly.",
  replaceWith = ReplaceWith("misk.redis.testing.FakeRedis")
)
```